subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/repositories_cheat_sheet.adoc

```diff
@@ -0,0 +1,425 @@
+[[repositories-cheat-sheet]]
+= Repositories Cheat Sheet
+
+[TIP]
+====
+Just getting started with repositories?
+Check out the <<declaring_repositories.adoc#declaring-repositories, Declaring Repositories>> to learn the basics of repositories.
+====
+
+[[sec:handling_credentials]]
+== Credentials
+
+Never store credentials in plain text in your build script.
+Instead, store the credentials in a local `gradle.properties` file and use the link:https://plugins.gradle.org/plugin/nu.studer.credentials[credentials plugin] to encrypt and consume credentials.
+
+You can use link:{javadocPath}/org/gradle/api/artifacts/repositories/AuthenticationSupported.html#credentials-java.lang.Class-[AuthenticationSupported.credentials]
+to limit the credentials used to access a repository to only the relevant credential types.
+Gradle looks up credentials from your <<build_environment.adoc#sec:gradle_configuration_properties,Gradle Properties>> when a build requires them.
+
+Gradle supports lookup of the following credential types:
```

Comment:
nitpick:
```suggestion
Gradle supports lookup for the following credential types:
```

Reply:
I can go either way on this. Neither is explicitly wrong, to my knowledge. Let me know if you feel especially strongly.